src/StructuredLogViewer.Core/ProjectGraph/RuntimeGraph.cs

```diff
@@ -11,6 +13,7 @@ namespace StructuredLogViewer.Core.ProjectGraph
 {
     public class RuntimeGraph
     {
+        [DebuggerDisplay("{ToString()}")]

```

Comment:
No need for this line at all now ;)

Reply:
I'll fix later np

Reply:
Ah, didn't know the Debugger calls ToString by default on the objects if they override it.